.github/workflows/test.yml

```diff
@@ -144,6 +144,55 @@ jobs:
           retention-days: 1
           if-no-files-found: warn
 
+  test-helm:
+    runs-on: ubuntu-latest
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0 # Shallow clones should be disabled for a better relevancy of analysis
+      - name: Set up Helm
+        uses: azure/setup-helm@v4.2.0
+        with:
+          version: v3.14.4
```

Comment:
is this the version we have installed or what's the significance of v3.14.4?

Reply:
removed